src/components/Blog.js

```diff
@@ -0,0 +1,17 @@
+function Post() {
```

Comment:
Maybe, it's okay to have it here while the components are small and used for prototype purposes, but better to keep them separate when the content becomes more meaningful.
It also related to the menu and header, news and article